publish-common/src/main/java/com/ericsson/eiffel/remrem/publish/helper/RabbitMqProperties.java

```diff
@@ -51,6 +52,8 @@ public class RabbitMqProperties {
     private String domainId;
     private Integer channelsCount;
     private boolean createExchangeIfNotExisting;
+    private Long waitForConfirmsTimeOut;
+    private Long defaultWaitForConfirmsTimeout = 5000L;
```

Comment:
Is it a constant? If so, should be ALL_UPPERCASE and declared as final static, as other constants.

Reply:
will change it